src/cljs/ataru/virkailija/editor/components/followup_question.cljs

```diff
@@ -28,19 +30,22 @@
       [:div.editor-form__followup-indicator]
       [:div.editor-form__followup-indicator-inlay]
       [:div.editor-form__followup-question-overlay
-       (into [:div] followups)
+       followups
        [toolbar/followup-toolbar option-path
         (fn [generate-fn]
           (dispatch [:editor/generate-followup-component generate-fn option-path]))]]]]))
 
-(defn followup-question [option-index followups option-path show-followups]
+(defn followup-question [option-index followups option-path show-followups parent-key option-value]
   (let [attrs {:on-click #(swap! show-followups
                                  (fn [v] (update v option-index not)))}]
     [:div.editor-form__followup-question
+     [copy-link (str parent-key "_" option-value) :answer? true]
```

Comment:
Tänne vois kans ujuttaa tiedon siitä onko kysymys kysymysryhmän sisällä. Esim. `custom-answer-options` näyttäis saavan sen tiedon jo parametrina. Niin sais oikein muotosen tunnisteen sillonkin, joskin siihen täytyy laittaa joku place holder sitä juoksevaa numeroa varten, tyyliin `(str parent-key "_group%d_" option-value)`.